docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,362 @@
+export CLUSTER_NAME=<CLUSTER_NAME>
+export AWS_DEFAULT_REGION=<CLUSTER_REGION>
+aws eks update-kubeconfig --name $CLUSTER_NAME --region $AWS_DEFAULT_REGION
+kubectl config get-contexts
+# Ensure cluster has compute
+kubectl get nodes
+```
+
+Before you can deploy your SageMaker service controller using an IAM role, associate an OpenID Connect (OIDC) provider with your IAM role to authenticate your cluster with the IAM service.
+
+```bash
+eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+--region ${AWS_DEFAULT_REGION} --approve
+```
+
+Get the following OIDC information for future reference:
+
+```bash
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export OIDC_PROVIDER_URL=$(aws eks describe-cluster --name $CLUSTER_NAME --region $AWS_DEFAULT_REGION \
+--query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+```
+
+In your working directory, create a file named `trust.json` using the following trust relationship code block:
+
+```bash
+printf '{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::'$AWS_ACCOUNT_ID':oidc-provider/'$OIDC_PROVIDER_URL'"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "'$OIDC_PROVIDER_URL':aud": "sts.amazonaws.com",
+          "'$OIDC_PROVIDER_URL':sub": [
+            "system:serviceaccount:ack-system:ack-sagemaker-controller",
+            "system:serviceaccount:ack-system:ack-applicationautoscaling-controller"
+          ]
+        }
+      }
+    }
+  ]
+}
+' > ./trust.json
+```
+
+Run the `iam create-role` command to create an IAM role with the trust relationship you just defined in `trust.json`. This IAM role enables the Amazon EKS cluster to get and refresh credentials from IAM.
+
+```bash
+export OIDC_ROLE_NAME=ack-controller-role-$CLUSTER_NAME
+aws --region $AWS_DEFAULT_REGION iam create-role --role-name $OIDC_ROLE_NAME --assume-role-policy-document file://trust.json
+```
+
+Attach the AmazonSageMakerFullAccess Policy to the IAM Role to ensure that your SageMaker service controller has access to the appropriate resources. 
+
+```bash
+aws --region $AWS_DEFAULT_REGION iam attach-role-policy --role-name $OIDC_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+export IAM_ROLE_ARN_FOR_IRSA=$(aws --region $AWS_DEFAULT_REGION iam get-role --role-name $OIDC_ROLE_NAME --output text --query 'Role.Arn')
+echo $IAM_ROLE_ARN_FOR_IRSA
+```
+
+For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
+
+### Install the SageMaker ACK service controller
+
+Get the SageMaker Helm chart and make it available on the `Deployment` host with the following commands:
+
+```bash
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=sagemaker
+export RELEASE_VERSION=v0.0.4
```

Comment:
Do we want to put a specific release version here? We might do something like this instead?

```
LATEST_RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/sagemaker-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
RELEASE_VERSION=${LATEST_RELEASE_VERSION:-v0.0.4}
```

Once Sagemaker releases a non-pre-prod release of sagemaker-controller, then the `RELEASE_VERSION` variable will contain that latest release version, otherwise it will be v0.0.4...